rkt/gc.go

```diff
@@ -67,6 +69,10 @@ func runGC(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
+	if flagMarkOnly {
```

Comment:
@jonboulle If we don't skip renameExpired, then when doing `gc --mark-only`, we need to give a very long time duration to `--expired-period`, otherwise it will remove some just prepared pods.
I would like `gc --mark-only` to be called without other flags.